tests/integration/test_services_auto_tagging_copilot_feedback.py

```
@@ -0,0 +1,1217 @@
+        key = "rename:.txt"
+        assert key in fb.pattern_adjustments
+
+    def test_get_user_history(self, tmp_path: Path) -> None:
+        from file_organizer.models.suggestion_types import Suggestion, SuggestionType
+        from file_organizer.services.suggestion_feedback import SuggestionFeedback
+
+        fb = SuggestionFeedback(feedback_file=tmp_path / "feedback.json")
+        s = Suggestion(
+            suggestion_id="h1",
+            suggestion_type=SuggestionType.MOVE,
+            file_path=Path("/tmp/doc.pdf"),
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/doc.pdf"

(S108)